chapters/ch32.md

```diff
@@ -1,49 +1,47 @@
 
 > * If your training set is skewed towards one class, or if it has many classes, choose a “balanced” subset instead of 10 training examples at random out of the set of 100. For example, you can make sure that 2/10 of the examples are positive examples, and 8/10 are negative. More generally, you can make sure the fraction of examples from each class is as close as possible to the overall fraction in the original training set.
 
---> _replace THIS LINE by your translation for the above line_
+* Nếu tập huấn luyện của bạn bị lệch về một lớp, hoặc nếu nó có nhiều lớp, hãy chọn một tập hợp con “cân bằng” thay vì 10 mẫu huấn luyện ngẫu nhiên trong số 100 mẫu. Ví dụ, bạn có thể chắc chắn rằng 2/10 các mẫu là các mẫu dương và 8/10 là âm. Tổng quát hơn, bạn có thể đảm bảo tỷ lệ các lớp trong tập mẫu càng gần với tỉ lệ trong tập huấn luyện ban đầu.
 
 
 > I would not bother with either of these techniques unless you have already tried plotting learning curves and concluded that the curves are too noisy to see the underlying trends. If your training set is large—say over 10,000 examples—and your class distribution is not very skewed, you probably won’t need these techniques.
 
---> _replace THIS LINE by your translation for the above line_
+Tôi sẽ không quan tâm với một trong những kỹ thuật này trừ khi bạn đã thử vẽ các đồ thị quá trình học và kết luận rằng các đường cong quá nhiễu để nhìn thấy các xu hướng cơ bản. Nếu tập huấn luyện của bạn có quy mô lớn, ví dụ hơn 10.000 mẫu, và phân phối lớp của bạn không bị lệch, có lẽ bạn không cần các kỹ thuật này.
 
 
 > Finally, plotting a learning curve may be computationally expensive: For example, you might have to train ten models with 1,000, then 2,000, all the way up to 10,000 examples. Training models with small datasets is much faster than training models with large datasets. Thus, instead of evenly spacing out the training set sizes on a linear scale as above, you might train models with 1,000, 2,000, 4,000, 6,000, and 10,000 examples. This should still give you a clear sense of the trends in the learning curves. Of course, this technique is relevant only if the computational cost of training all the additional models is significant.
 
---> _replace THIS LINE by your translation for the above line_
+Cuối cùng, vẽ đồ thị quá trình học có thể tốn kém về mặt tính toán: Ví dụ: bạn có thể phải huấn luyện mười mô hình với 1.000, rồi 2.000, cho đến 10.000 mẫu. Huấn luyện các mô hình với các bộ dữ liệu nhỏ nhanh hơn nhiều so với huấn luyện các mô hình với các bộ dữ liệu lớn. Do đó, thay vì cách đều các kích thước tập huấn luyện theo tỷ lệ tuyến tính như trên, bạn có thể huấn luyện các mô hình với 1.000, 2.000, 4.000, 6.000 và 10.000 mẫu. Điều này vẫn sẽ cung cấp cho bạn một cảm giác rõ ràng về các xu hướng trong các đồ thị quá trình học. Tất nhiên, kỹ thuật này chỉ có liên quan nếu chi phí tính toán để huấn luyện tất cả các mô hình bổ sung là đáng kể.
```

Comment:
- "relevant" ở đây nên dịch là "phù hợp", "thích hợp", "xác đáng", (nói chung là = "applicable"), chứ không phải là "liên quan" (kiểu như "related to")

Reply:
ok bạn. Dịch như vậy rõ nghĩa hơn. Mình chọn từ "thích hợp".

Reply:
> thay vì cách đều các kích thước tập huấn luyện theo tỷ lệ tuyến tính như trên

Có thể viết như này được không ta @@

> Thay vì phải chia kích thước của tập huấn luyện theo cấp số cộng với công sai là 1000 như ở trên